### PR TITLE
Kfspts 7968 pdp xml pojo

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
Sure, I can change the class names.  Eclipse has a jaxb tool to create XML pojos from an XSD file.  The tool didn't like a couple things in the payment.xsd file so I had to work around a few issues though.

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
This is ready for review again

**Comment by db483** [comment]
Hi Jay,

Would you mind sharing a bit about how you generated these classes at the next bookclub? Thank you.

**Comment by jhulslander** [comment]
sure thing

**Comment by jhulslander** [comment]
rettest this please

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
This is ready for review again, the unit tests are passing in Jenkins

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
I made the namespace a constant, I also moved some test data creation into a fixture.

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
this is ready for review again

File: src/test/java/edu/cornell/kfs/concur/batch/xmlObjects/TestPdpMarshal.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        FileUtils.deleteDirectory(batchDirectoryFile);
+    }
+
+    @Test
+    public void test() throws JAXBException, IOException, SAXException, ParserConfigurationException {
+        PdpFeedFileBaseEntry pdpFile = PdpFeedFileBaseEntryFixture.buildPdpFile();
```
Comment (db483) [inline src/test/java/edu/cornell/kfs/concur/batch/xmlObjects/TestPdpMarshal.java]: I think both batch jobs will need to call code similar to this to generate the xml file. Should this be extracted in a service that could be reused? Thank you.

Comment (jhulslander) [inline src/test/java/edu/cornell/kfs/concur/batch/xmlObjects/TestPdpMarshal.java]: OK, I shall movee to a service

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
this is ready for review again

**Comment by jhulslander** [comment]
woot, thank you

**Pull request merged**